### PR TITLE
Admin import in Django admin

### LINKED ISSUE #169: Admin action for running importers
At least sync operations for tprek and kirjastot.fi opening times

### PR DESCRIPTION
**Pull request by otiala**
Closes #169 

Added import functionalities to Django admin for Unit-model and Municipality-model.

Municipalities can't be removed from the Django admin.

Clicking the import buttons on said models will perform `call_command()` with certain parameters, capture it's output, redirect to a new admin page where said output will be shown.

### CODE REVIEW

File: resources/admin/__init__.py [hunk 1]
```diff
@@ -204,6 +244,51 @@ class ResourceGroupAdmin(PopulateCreatedAndModifiedMixin, CommonExcludeMixin, Fi
...
+        context['opts'] = self.model._meta
+        return TemplateResponse(request, self.import_template, context)
+
+    def divisions_helsinki_import(self, request):
+        context = dict(
+            self.admin_site.each_context(request),
+        )
+        out = StringIO()
+        with redirect_stdout(out):
+            call_command('geo_import', '--divisions', 'helsinki', stdout=out)
```
Comment (jussih) [inline resources/admin/__init__.py]: `try`-block here as well

File: resources/admin/__init__.py [hunk 2]
```diff
@@ -204,6 +244,51 @@ class ResourceGroupAdmin(PopulateCreatedAndModifiedMixin, CommonExcludeMixin, Fi
...
+    def get_actions(self, request):
+        actions = super().get_actions(request)
+        if 'delete_selected' in actions:
+            del actions['delete_selected']
+        return actions
+
+    def get_urls(self):
+        urls = super(MunicipalityAdmin, self).get_urls()
+        extra_urls = [
+        url(r'^municipalities_import/$', self.admin_site.admin_view(self.municipalities_import), name='municipalities_import'),
```
Comment (jussih) [inline resources/admin/__init__.py]: Indentation

**Comment by jussih** [comment]
The `codecov/project` check should stop complaining after this is either rebased on top of master or master is merged to this. The complaint comes because master currently has lower coverage due to kulkunen.

File: resources/admin/__init__.py [hunk 3]
```diff
@@ -219,3 +328,4 @@ class ResourceGroupAdmin(PopulateCreatedAndModifiedMixin, CommonExcludeMixin, Fi
 admin_site.register(ReservationMetadataField)
 admin_site.register(ReservationMetadataSet, ReservationMetadataSetAdmin)
 admin.site.register(ResourceGroup, ResourceGroupAdmin)
+admin.site.register(Municipality, MunicipalityAdmin)
```
[the remark this answers is not in the document]
Comment (otiala) [reply resources/admin/__init__.py]: Good catch @jussih. I added a check in a new commit.

**Pull request merged**